src/components/parts/commons/Modal/Modal.tsx

```diff
@@ -27,10 +27,21 @@ export const Modal: VFC<Props> = ({ open, emojiId, title, content, onClose, size
     <MuiModal open={open} onClose={onClose}>
       <StyledBox width={sizeMap[size]}>
         <Box py="8px" display="flex" alignItems="center" justifyContent="center" alignContent="center" gap="8px">
-          {emojiId && <Emoji emojiId={emojiId} size={24} />}
-          <Typography display="block" variant="h4">
-            {title}
-          </Typography>
+          {size === 'small' ? (
+            <>
+              {emojiId && <Emoji emojiId={emojiId} size={20} />}
+              <Typography display="block" variant="body1">
+                {title}
+              </Typography>
+            </>
+          ) : (
+            <>
+              {emojiId && <Emoji emojiId={emojiId} size={24} />}
+              <Typography display="block" variant="h4">
+                {title}
+              </Typography>
```

Comment:
好みの話なのでどちらでもですが

```suggestion
              {emojiId && <Emoji emojiId={emojiId} size={size === 'small'? 20 ? 24 } />}
              <Typography display="block" variant={size === 'small' ? "body1": "h4">
                {title}
              </Typography>
```
の方が好きですね